pycrdt_websocket/django_channels/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .yjs_consumer import YjsConsumer as YjsConsumer
+from .yroom_storage import BaseYRoomStorage as BaseYRoomStorage
+from .yroom_storage import RedisYRoomStorage as RedisYRoomStorage
```

Comment:
Does this mean that redis is a required dependency when using Django channels?

Reply:
Oh, right. I've removed `RedisYRoomStorage` from this init file and separate them in files so when you import the base class you are not requiring redis. Maybe we should do the same with `sqlite-anyio` on YStore, which is a dependency for the whole project.